### PR TITLE
Test page created

### PR DESCRIPTION
**Pull request by isralduke**
This PR:

- Creates a test page to visually check CSS changes correctness
- Creates a new layout page
- Corrects some CSS typos

### CODE REVIEW

File: docs/_layouts/test-page.njk [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: base.njk
+---
+
+<!-- this style is only for the test page -->
+<style scoped>
+  .toast         {
+    position: fixed;
+    right: 10px;
+    z-index: 3000;
```
Comment (frogpad7) [inline docs/_layouts/test-page.njk:10]: If the toasts are meant to be underneath the modal, I believe the range of the z-index would fall into the layer before modal being less than 1040.

File: docs/test-page/index.md [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+        <hr class="minor">
+      </div>
+    </div>
+    {%- include 'markup/button-page-actions.njk' -%}
+  </div>
+</div>
+
+<!-- modals and toasts buttons -->
+
+<div class="content-container">
```
Comment (frogpad7) [inline docs/test-page/index.md:240]: If this were a btn-container, the buttons would space better at mobile sizes. The specificity of the button spacing needs both the .btn-container and the .btn classes.
```suggestion
<div class="btn-container">
```

File: docs/test-page/index.md [hunk 3]
```diff
@@ -0,0 +1,362 @@
...
+    <div class="col-12 col-lg-4">
+      <div class="data-tile">
+        <div class="data-tile-body">
+          <p class="fw-bold text-black mb-0">Label</p>
+          <hr class="minor my-2">
+          <p class="mb-0">Value</p>
+        </div>
+      </div>
+      {%- include 'markup/progress-small.njk' -%}
+    </div>
```
Comment (frogpad7) [inline docs/test-page/index.md:281]: There is no bottom margin spacing at mobile sizes between the progress-small and the share-social partials.

**Review by frogpad7 — CHANGES_REQUESTED** [review]
A few things I found while at a mobile sized screen; Also curious about the toast positioning, especially with the z-index.

File: docs/_layouts/test-page.njk (hunk 1 above)
Comment (isralduke) [reply docs/_layouts/test-page.njk:10]: i am not worried about that on this test page

File: docs/test-page/index.md (hunk 3 above)
Comment (isralduke) [reply docs/test-page/index.md:281]: good catch

File: docs/test-page/index.md (hunk 2 above)
Comment (isralduke) [reply docs/test-page/index.md:240]: Not worried about it here

**Review by frogpad7 — APPROVED** [review]
Test page is now ready for testing.

**Review by frogpad7 — APPROVED** [review]
good semantic small updates.

**Review by rltucker — CHANGES_REQUESTED** [review]
test-page/index.html -- Spelling Error 'visuall' should be 'visually'

**Review by rltucker — APPROVED** [review]
Approved

**Review by james-alt — APPROVED** [review]
:shipit:

**Pull request merged**